CliWrap/Cli.cs

```diff
@@ -41,9 +42,9 @@ public Cli(string filePath)
         {
         }
 
-        private Process CreateProcess(string arguments)
+        private Process CreateProcess(string arguments, Dictionary<string, string> env = null)
```

Comment:
At this point, just pass ExecutionInput instead of two parameters.

Reply:
Yeah I was gonna do that, thought you might prefer it this way,